Adding references to the source of copied files

DucAnh-silabs opened the pull request:
Adding references to the source of files that have been taken from zipgateway

rzr commented on README.md:
```diff
@@ -191,3 +191,7 @@ commander flash nvm.bin --address=0x00074000
+
+## References
+
+* https://github.com/SiliconLabs/ZIPGATEWAY
```
```suggestion
* https://github.com/SiliconLabs/zipgateway (upstream source of duplicated files)
```

DucAnh-silabs replied:
Okay

DucAnh-silabs committed:
Adding references to the source of duplicated files:
* CMakeLists.txt
* zw_nvm_converter/CMakeLists.txt
* zw_nvm_converter/controllerlib_7.xx/CMakeLists.txt
* zw_nvm_converter/controllerlib_7.xx/api_wrappers.c
* zw_nvm_converter/controllerlib_7.xx/controller_nvm.c
* zw_nvm_converter/controllerlib_7.xx/controller_nvm.h
* zw_nvm_converter/controllerlib_7.xx/endian_wrap.h
* zw_nvm_converter/controllerlib_7.xx/nvm3_helpers.c
* zw_nvm_converter/controllerlib_7.xx/nvm3_helpers.h
* zw_nvm_converter/controllerlib_7.xx/testfiles/hex2bin
* zw_nvm_converter/controllerlib_7.xx/testfiles/zg14_711/001_FBFEDD24.xml
* zw_nvm_converter/controllerlib_7.xx/testfiles/zg14_711/ControllerNVMBackup.bin
* zw_nvm_converter/controllerlib_7.xx/testfiles/zg14_712/ControllerNVMBackup.bin
* zw_nvm_converter/controllerlib_7.xx/testfiles/zgm130s_711/001_D9545A3C.xml
* zw_nvm_converter/controllerlib_7.xx/testfiles/zgm130s_711/ControllerNVMBackup.bin
* zw_nvm_converter/controllerlib_7.xx/zwave/ZW_nodemask.c
* zw_nvm_converter/controllerlib_7.xx/zwave/ZW_nodemask_api.h
* zw_nvm_converter/controllerlib_7.xx/zwave/ZW_transport_api_small.h
* zw_nvm_converter/controllerlib_7.xx/zwave/zwave_controller_application_info_storage.h
* zw_nvm_converter/controllerlib_7.xx/zwave/zwave_controller_network_info_storage.h
* zw_nvm_converter/examples/protocol_example.json
* zw_nvm_converter/include/controllerlib_api.h
* zw_nvm_converter/include/user_message.h
* zw_nvm_converter/libs/CMakeLists.txt
* zw_nvm_converter/libs/app_utils/CMakeLists.txt
* zw_nvm_converter/libs/app_utils/base64.c
* zw_nvm_converter/libs/app_utils/base64.h
* zw_nvm_converter/libs/app_utils/user_message.c
* zw_nvm_converter/libs/app_utils/user_message.h
* zw_nvm_converter/libs/bin2hex/CMakeLists.txt
* zw_nvm_converter/libs/bin2hex/bin2hex.c
* zw_nvm_converter/libs/bin2hex/bin2hex.h
* zw_nvm_converter/libs/json_helpers/CMakeLists.txt
* zw_nvm_converter/libs/json_helpers/endian_wrap.h
* zw_nvm_converter/libs/json_helpers/json_helpers.c
* zw_nvm_converter/libs/json_helpers/json_helpers.h
* zw_nvm_converter/nvmconv_main.c
* zw_nvm_converter/nvmconv_main.h
These files were taken from
(zipgateway)[https://github.com/SiliconLabs/ZIPGATEWAY] project and
modified.
Signed-off-by: Duc Anh Nguyen <annguyen5@silabs.com>
